build/build/src/engine/context.rs

```diff
@@ -686,7 +687,25 @@ pub async fn runner_sanity_test(
             .run_ok()
             .await;
 
-        test_base.and(test_internal_base).and(test_geo)
+        let all_cmds = test_base.and(test_internal_base).and(test_geo);
+
+        // The following test does not actually run anything, it just checks if the engine
+        // can accept `--jvm` argument and evaluates something.
+        if TARGET_OS != OS::Windows {
```

Comment:
When the test runs on the CI, there is nothing inside `ENSO_DATA_DIRECTORY`. The `--jvm` argument first tries to find any appropriate JDK in the `ENSO_DATA_DIRECTORY/runtime` directory, but on the CI, it is empty. The fallback is to search in `JAVA_HOME` env var or on `PATH`, but that works only on Linux. Let's just keep this sanity check on Linux for now.

Reply:
Trying to run the tests also on Windows again in https://github.com/enso-org/enso/pull/11500/commits/fb25d57ed8cad3f563489440353444eaec4d6706

Reply:
So that didn't work?